intel-retail/rtsf-at-checkout-reference-design
# feat(device-scale): Migrate device-scale device service to EdgeX v3.0

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**74**
closes: #74 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- Run `make device-scale` 
- Comment out `event-reconciler`, `product-lookup` and `loss-detector` in `docker-compose.loss-detection.yml`
- Run `make run-portainer run-base`
- Navigate to `localhost:9000`
- Check `device-scale` portainer logs 
    - Note: `device-scale` will have a warning/error about not finding the serial port and/or the scale is not connected. (but that is okay)

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/rtsf-at-checkout-reference-design )
